<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turbine Dauertest V1.7</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>404</width>
      <height>438</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pbConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbPorts">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Disconnect">
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pb_Start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pb_test">
       <property name="text">
        <string>Start Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_Stop">
       <property name="text">
        <string>StopTest</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_Rpm"/>
     </item>
     <item>
      <widget class="QSlider" name="sliderRPM">
       <property name="minimum">
        <number>3000</number>
       </property>
       <property name="maximum">
        <number>36000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="pageStep">
        <number>5000</number>
       </property>
       <property name="value">
        <number>6004</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Error">
       <property name="text">
        <string>ERROR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Rpm">
       <property name="text">
        <string>RPM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Strom">
       <property name="text">
        <string>Strom</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Volt">
       <property name="text">
        <string>Volt</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Druck">
       <property name="text">
        <string>Druck</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Version">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="l_Tc">
       <property name="text">
        <string>TestCount</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="l_StatusLine">
         <property name="text">
          <string>Status Line</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
